cardano-api/src/Cardano/Api/Script.hs

```diff
@@ -104,60 +104,53 @@ module Cardano.Api.Script (
 
 import           Prelude
 
+import           Control.Applicative
+import           Control.Monad
+import           Data.Aeson (Value (..), object, (.:), (.=))
+import qualified Data.Aeson as Aeson
+import qualified Data.Aeson.Types as Aeson
 import qualified Data.ByteString.Lazy as LBS
 import           Data.ByteString.Short (ShortByteString)
 import qualified Data.ByteString.Short as SBS
 import           Data.Foldable (toList)
 import           Data.Scientific (toBoundedInteger)
+import qualified Data.Sequence.Strict as Seq
 import           Data.String (IsString)
 import           Data.Text (Text)
 import qualified Data.Text as Text
 import qualified Data.Text.Encoding as Text
 import           Data.Type.Equality (TestEquality (..), (:~:) (Refl))
 import           Data.Typeable (Typeable)
-import           Numeric.Natural (Natural)
-
-import           Data.Aeson (Value (..), object, (.:), (.=))
-import qualified Data.Aeson as Aeson
-import qualified Data.Aeson.Types as Aeson
-import qualified Data.Sequence.Strict as Seq
 import           Data.Vector (Vector)
 import qualified Data.Vector as Vector
-
-import           Control.Applicative
-import           Control.Monad
+import           Numeric.Natural (Natural)
 
 import qualified Cardano.Binary as CBOR
-
 import qualified Cardano.Crypto.Hash.Class as Crypto
-
-import           Cardano.Slotting.Slot (SlotNo)
-
+import qualified Cardano.Ledger.Alonzo.Language as Alonzo
+import qualified Cardano.Ledger.Alonzo.Scripts as Alonzo
 import           Cardano.Ledger.BaseTypes (StrictMaybe (..))
 import qualified Cardano.Ledger.Core as Ledger
 import qualified Cardano.Ledger.Era as Ledger
-
 import qualified Cardano.Ledger.Keys as Shelley
 import qualified Cardano.Ledger.Shelley.Scripts as Shelley
 import qualified Cardano.Ledger.ShelleyMA.Timelocks as Timelock
+import           Cardano.Slotting.Slot (SlotNo)
 import           Ouroboros.Consensus.Shelley.Eras (StandardCrypto)
-
-import qualified Cardano.Ledger.Alonzo.Language as Alonzo
-import qualified Cardano.Ledger.Alonzo.Scripts as Alonzo
-
 import qualified Plutus.V1.Ledger.Examples as Plutus
 
 import           Cardano.Api.Eras
-import           Cardano.Api.Error
-import           Cardano.Api.HasTypeProxy
+import           Cardano.Api.Error (displayError)
 import           Cardano.Api.Hash
+import           Cardano.Api.HasTypeProxy
```

Comment:
Do you really need to reorder all these imports? These are changes unrelated to the purpose of the PR.